doc/release_notes.rst

```diff
@@ -5,6 +5,13 @@ Release Notes
 0.2 series
 ..........
 
+0.2.2 (2021-03-19)
+------------------
+
+* Add checks in SBML conversion (e.g. to catch invalid identifiers and equations).
```

Comment:
optionally add (# commit id)

Reply:
Thought about it, but was not done so far, so I preferred to be consistent :)

Reply:
Might be something to add also for v0.2.1 at some point for the next release :)

Reply:
Ok, changed my mind and added them :D